lib/ansible/modules/system/authorized_key.py

```diff
@@ -107,9 +107,9 @@
     user: deploy
     state: present
     key: '{{ item }}'
```

Comment:
or instead make this line: `"{{ lookup('file', item) }}"` and leave the items below alone